src/init.cpp

```diff
@@ -493,7 +493,7 @@ std::string HelpMessage(HelpMessageMode mode)
 
     strUsage += HelpMessageGroup(_("Block creation options:"));
     strUsage += HelpMessageOpt("-blockminsize=<n>", strprintf(_("Set minimum block size in bytes (default: %u)"), DEFAULT_BLOCK_MIN_SIZE));
-    strUsage += HelpMessageOpt("-blockmaxsize=<n>", strprintf(_("Set maximum block size in bytes (default: %d)"), DEFAULT_BLOCK_MAX_SIZE));
+    strUsage += HelpMessageOpt("-blockmaxsize=<n>", _("Set maximum block size in bytes (default is equal to network max-block-size)"));
```

Comment:
I contemplated that one, didn't do it because the bitcoind may have no blocks yet, or it may be behind and it may run a long time. In all cases it would potentially say 1MB while the effective turns into 2MB all of a sudden.  Best to actually say so and not let that be a surprise.